README.md

```diff
@@ -95,6 +100,7 @@ write close
 ```
 # accept new bidirectional streams at "receiver" from source "sender"
 # for route reference ${routeRef} (required)
+# with specified authorization (optional, default zeros, exact match required)
```

Comment:
Exact match seems overly strict, no?

Reply:
Exact matches the only way to verify what comes out the other side of the nukleus is correct. Given a particular input stream with a particular authorization there is only one correct answer out the other side. I realized that without exact match I cannot verify correct behavior of the security nukleus for example, or even any nukleus, given that all of the other nuklei should pass on the authorization bits unmodified.

Reply:
I understand, thanks for clarifying.